### ISSUE TITLE
Creation of download ticket times out (>60s) when creating archive for large data set

### ISSUE THREAD

**Issue by nagem** [flywheel-io/core#1124]
The `api/download` endpoint has a two state process:
 1. Ticket Request 
  - the user makes a request containing the containers they would like to download
  - the API creates a proposed download object containing paths and other information about each file in the requested containers, as well as overall download size and file count and returns this information to the user with a ticket id

2. Download Archive
  - the user makes another request with the ticket id
  - the API streams the archive to the user as it is built

The first part of the request seems to be timing out (lasting longer than 60s) while attempting to build the listing of files and their expected paths. This is affecting users trying to download projects with >540 sessions.

**Comment by AshleyAncona** [comment]
Comment from Implementations meeting: 
- need to start by fixing the html response so that Dan can render it a proper error
- Give user warning when uploading 100 that it may be too much

**Comment by nagem** [comment]
The first bullet point can be addressed with nginx settings and should be moved to fly/fly, the second would have to be a client level warning and should be moved to fly/frontend (and I think you mean download?)

**Comment by ryansanford** [comment]
@nagem regarding nginx settings, you are referring to the body of the http 504 response?

**Comment by nagem** [comment]
@ryansanford yes, I don't have any control over what the body of the response looks like if nginx times out the request. I think @kofalt was looking in to a change that would give more information in these types of cases.
